src/main/java/frc/robot/Robot.java

```diff
@@ -28,15 +32,20 @@
   public static double[] changeAngle = new double[]{};
   public static double[] changeRotations = new double[]{};
   public static boolean Shift;
+  public static DoubleSolenoid lShifter;
+  public static DoubleSolenoid rShifter;
   int i;
 
   @Override
   public void robotInit() {
+    lShifter = new DoubleSolenoid(0, 7);
+    rShifter = new DoubleSolenoid(1, 6);
     autoTime = new Timer();
     Drive.initializeDrive();
     //Ballmech.initializeBallMech();
     autonomousCommand = new TestDrive();
     Constants.constantInit();
+    Drive.Climber();
```

Comment:
Does drive really have a climber? Should this be its own class with an init method and telop method?